core/noah_core/blueprints/interactions/information_extraction/answers.py

```diff
@@ -0,0 +1,26 @@
+import os
+from dotenv import load_dotenv
+from langchain.chains import RetrievalQA
+from langchain.embeddings.openai import OpenAIEmbeddings
+from langchain.llms import OpenAI
+from langchain.vectorstores import Milvus
+
+
+load_dotenv('.env')
+
+
+def answer(query):
+  docsearch = Milvus(
+    embedding_function=OpenAIEmbeddings(),
+    collection_name="messages",
+    connection_args={
+      "uri": os.environ['ZILLIZ_ENDPOINT'],
+      "token": os.environ['ZILLIZ_TOKEN'],
+    },
+  )
+
+  qa = RetrievalQA.from_chain_type(llm=OpenAI(), chain_type="stuff", retriever=docsearch.as_retriever())
```

Comment:
Just to know, is this 'stuff' a specific type? or is it a default value?

Reply:
It is a type of behavior of a RetrievalQA chain, explanation here https://python.langchain.com/docs/modules/chains/additional/question_answering.html